spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -1763,6 +1771,71 @@ public void testAllManifestTableSnapshotFiltering() throws Exception {
     }
   }
 
+  @Test
+  public void testTableWithInt96Timestamp() throws IOException {
+    try {
```

Comment:
do we need the try-catch here? I think it would work fine without it

Reply:
There is only the finally method here, it does a cleanup operation and I imitate what other UT wrote